unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -320,46 +403,68 @@ bool IsInsideTheLimits(SceneObject sceneObject)
 
         if (limits.bodies < usage.bodies + sceneObject.metrics.bodies)
         {
-            HUDController.i.buildModeHud.ShowSceneLimitsPassed();
+            HUDController.i.builderInWorldMainHud?.ShowSceneLimitsPassed();
```

Comment:
Maybe we can check for `HUDController.i.builderInWorldMainHud` nullity ahead on time and `return false;` to avoid all these collapsed nullity checks?